rs-react-app/src/redux/formSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface UserData {
```

Comment:
```suggestion
  interface UserData extends Omit<FormData, 'confirmPassword'> {
  picture: string;
}
```
or at least use `Gender` enum